installer-howto/installer-howto.rst

```diff
@@ -0,0 +1,246 @@
+.. _installer_howto:
+
+Rockstor’s “Built on openSUSE” installer - Beta release
+=======================================================
+
+.. _installer_thanks:
+
+Thanks
+------
+
+Rockstor’s developers would like to thank the entire OSS community for making any of this possible.
+Specifically with our fledgling move / rebase on openSUSE Leap15.2 our thanks go to the openSUSE/SuSE organisations
+and the larger community for the extreme generosity we all benefit from in a myriad of seen and unseen ways.
+
+**Please note: Rockstor is in no way affiliated with openSUSE / SuSE and does not wish, under any circumstances, to misrepresent these organisations.**
+*Any errors you encounter are most likely our fault and we try, on a current volunteer basis, to support our users via our* `friendly forum: <https://forum.rockstor.com/.>`_.
```

Comment:
It seems you intended on making the url of the forum displayed, but it isn't in my case. I thus see a trailing ":" that is undesirable. We may want to remove that ":".

Reply:
@FroggyFlox I think we are OK, within our 'official docs' with not showing the URL, especially when it's to our own domain.  This was, as you suspected, just a stray colon as I was battling a url embedded in an italic block. We use this same "friendly forum" / "communit forum" link elsewhere in the docs and on the main web page. So I've, as suggested pushed a commit (3b77831) to just remove the stray colon.

Thanks for catching this. Much appreciated. We could probably draw up some guidelines for when to show url as link text.